punchcut/src/lib.rs

```diff
@@ -0,0 +1,200 @@
+/*!
+Glyph loading.
+*/
+
+#![forbid(unsafe_code)]
+// TODO: this is temporary-- remove when hinting is added.
+#![allow(dead_code, unused_imports, unused_variables)]
+
+/// Re-export of peniko crate.
+pub use peniko;
```

Comment:
Related to previous comment in `Cargo.toml`, could we make the peniko dependency optional?

Reply:
As mentioned above, we can remove it, but future COLRv1 support will end up duplicating types from these libraries. FWIW, we own both peniko and kurbo.